.github/workflows/ci.yaml

```diff
@@ -47,6 +47,8 @@ jobs:
         shell: bash
         run: |
           pip install schemachange
```

Comment:
Did we want to lock down a schemachange version as well?

Reply:
Done!